doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,421 @@
+is critical. That key principle motivates many of the more detailed principles
+given below for what to pay for and whom to pay.
+
+What to pay for
+```````````````
+
+1. Pay for things that are important *and* otherwise won't get done.
+   *Rationale: there is way more to be done than there are funds to do all
+   those things. So count on interested volunteers or external sponsored work
+   to do many of those things.*
+2. Plan for sustainability. Don't rely on money always being there.
+3. Consider potential positive benefits for NumPy maintainers and contributors,
+   maintainers of other projects, end users, and other stakeholders like
+   packagers and educators.
+4. Think broadly. There's more to a project than code: websites, documentation,
+   community building, governance - it's all important.
+5. For proposed funded work, include paid time for others to review your work
+   if such review is expected to be significant effort - do not just increase
+   the load on volunteer maintainers.
+   *Rationale: we want the effect of spending funds to be positive for
+   everyone, not just for the people getting paid. This is also a matter of
+   fairness.*
+
+When considering development work, principle (1) implies that priority should
+be giving to (a) the most boring/painful tasks that no one likes doing, and to
+necessary structural changes to the code base that are too large to be done by
+a volunteer in a reasonable amount of time.
+
+There are also a large amount of tasks, activities, and projects outside of
+development work that are important and could enhance the project or community
+- think for example of user surveys, translations, outreach, dedicated
+mentoring of newcomers, community organization, website improvements, and
+administrative tasks.
+
+Time of people to perform tasks is also not the only thing that funds can be
+used for: expenses for in-person developer meetings or sprints, hosted hardware
+for benchmarking or development work, and CI or other software services could
+all be good candidates to spend funds on.
+
+Whom to pay
+```````````
+
+1. All else being equal, give preference to existing maintainers/contributors.
+2. Consider this an opportunity to make the project more diverse.
+3. Pay attention to the following when considering paying someone:
+
+   - the necessary technical or domain-specific skills to execute the tasks,
+   - communication and self-management skills,
+   - experience contributing to and working with open source projects.
+
+It will likely depend on the project/tasks whether there's already a clear best
+candidate within the NumPy team, or whether we look for new people to get
+involved. Before making any decisions, the decision makers should think about
+whether an opportunity should be advertised to give a wider group of people a
+chance to throw their hat in the ring for it.
+
+Compensating fairly
+```````````````````
```

Comment:
Given the amount of discussion, let's put in a `.. note::` here that this section will not be final until we have tried these guidelines at least 2-3 times and are happy with them (and if not, we adjust).